src/test/scala-2.11/markets/participants/TestMarketParticipant.scala

```diff
@@ -27,8 +27,10 @@ import scala.collection.mutable
 
 class TestMarketParticipant extends MarketParticipant {
 
-  val markets = mutable.Map.empty[Tradable, (ActorRef, Agent[Tick])]
+  val markets = mutable.Map.empty[Tradable, ActorRef]
 
   val outstandingOrders = mutable.Set.empty[Order]
 
+  val tickers = mutable.Map.empty[Tradable, Agent[Tick]]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1653013747/issues/source?bid=2921361&fileBranchId=3037089#l34)